src/script/conversation/ConversationRepository.ts

```diff
@@ -1927,6 +1940,14 @@ export class ConversationRepository {
     return this.pushToReceivingQueue(eventJson, eventSource);
   };
 
+  private async handleWipeMLSConversation(conversationId: QualifiedId) {
+    try {
+      this.core.service!.conversation.wipeMLSConversation(conversationId);
```

Comment:
So what we could do here is pass on a `groupId` instead of a `conversationId` to the core and avoid mapping convId to GroupId in the core. 

Here you could try to find the conversation in the database (this will give you the groupId). 
If the conversation doesn't exist in the database it's highly likely that it's also not on core and doesn't need wiping. 

WDYT?

Reply:
the gist of it would be 

```js
const conv = this.findConversation(qualifiedId);
if (conv) {
  core.wipe(conv.groupId);
}
```

Reply:
Good idea, done it ;)